app/konnect/org-management/audit-logging/verify-signatures.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Verify audit log signatures
+content_type: how-to
+badge: enterprise
+beta: true
+---
+
+{{site.konnect_short_name}} uses an [ED25519 signature](https://ed25519.cr.yp.to/) on the audit logs it produces.
+
+Audit logs can be exported in two different formats, CEF and JSON. 
+Calculating the signature is slightly different for these formats.
+
+## Verify a signature
+
+1. Retrieve the public key from the audit log JWKS endpoint:
+
+    ```sh
+    curl -i -X GET https://us.api.konghq.com/v2/audit-log-webhook/jwks.json \
+    --header "Authorization: Bearer TOKEN"
+    ```
```

Comment:
This endpoint does not require authorization. (Is publicly accessible)

```suggestion
    ```sh
    curl -i -X GET https://us.api.konghq.com/v2/audit-log-webhook/jwks.json
    ```
```